src/main/java/com/andrzejewski/todolist/StringService.java

```diff
@@ -0,0 +1,63 @@
+package com.andrzejewski.todolist;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Random;
+
+
+@Service
+public class StringService {
+
+    @Autowired
+    private StringRepository stringRepository;
+
+    public StringService(StringRepository stringRepository) {
+        this.stringRepository = stringRepository;
+    }
+
+    public List<StringEntity> getAllStrings() {
+        return stringRepository.findAll();
+    }
+
+    public StringEntity getById(Long id) {
+        StringEntity stringEntity = stringRepository.findById(id).orElseThrow(() ->
+                new IllegalStateException("String with that id doesn't exists"));
```

Comment:
You should create a custom exception type for that and handle it in custom `@ControllerAdvice` error handler